Tasks/PipAuthenticateV1/task.json

```diff
@@ -42,6 +42,27 @@
         "MultiSelectFlatList": "True"
       }
     },
+    {
+      "name": "workloadIdentityServiceConnection",
```

Comment:
Do we want to support ONLY WIF service connections? Should it be possible to use other types of ARM service connections?

Reply:
at this point yes, this is the design we have been working with. Happy to discuss further offline